modules/apps/organizations/organizations-item-selector-web/src/main/java/com/liferay/organizations/item/selector/web/internal/display/context/OrganizationItemSelectorViewDisplayContext.java

```diff
@@ -74,17 +78,31 @@ public SearchContainer<Organization> getSearchContainer()
 		OrganizationSearchTerms organizationSearchTerms =
 			(OrganizationSearchTerms)_searchContainer.getSearchTerms();
 
+		LinkedHashMap<String, Object> organizationParams =
+			LinkedHashMapBuilder.<String, Object>put(
+				"organizationsTree",
+				() -> {
+					int currentUserId = GetterUtil.getInteger(
```

Comment:
to get the userId probably is better something like this `_portal.getUserId(_liferayPortletRequest)` because it checks also "impersonated users".
Unfortunately you need to pass the _portal in the constructor of the DisplayContext (but it should be used only in one place so should be quickly).

In this case don't save it in a variable but inline it where used (and probably you don't need to have the () -> {} anymore because it will be one line